python-checks/src/main/java/org/sonar/python/checks/TooManyLinesInFileCheck.java

```diff
@@ -57,7 +60,7 @@ public void visitNode(AstNode astNode) {
 
     if (lines > maximum) {
       String message = "File \"{0}\" has {1} lines, which is greater than {2} authorized. Split it into smaller files.";
-      getContext().createFileViolation(this, message, getContext().getFile().getName(), lines, maximum);
+      addFileIssue(MessageFormat.format(message, getContext().pythonFile().file().getName(), lines, maximum));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "file"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)